x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/WaterfallItem.tsx

```diff
@@ -96,7 +96,9 @@ function PrefixIcon({ item }: { item: IWaterfallItem }) {
   }
 
   // icon for RUM agent transactions
-  const isRumAgent = item.transaction.agent.name === 'js-base';
+  const isRumAgent = ['js-base', 'rum-js'].includes(
+    item.transaction.agent.name
+  );
```

Comment:
I feel this should be a helper method `isRumAgent` - perhaps even colocated with the `AgentName` definition in APMDocs. What do you think?